lib/react_client.dart

```diff
@@ -217,6 +218,19 @@ final ReactDartInteropStatics _dartInteropStatics = (() {
     component.transferComponentState();
   }
 
+  void _callSetStateCallbacks(Component component) {
+    component.setStateCallbacks.forEach((callback()) { callback(); });
```

Comment:
Not really, that just types it to a function that take no params and returns dynamic.